src/main/java/de/qabel/desktop/ui/about/aboutPopup/AboutPopupController.java

```diff
@@ -0,0 +1,35 @@
+package de.qabel.desktop.ui.about.aboutPopup;
+
+import de.qabel.desktop.ui.AbstractController;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.TextArea;
+import javafx.scene.layout.Pane;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class AboutPopupController extends AbstractController implements Initializable {
+
+    @FXML
+    private TextArea textAreaPopup;
+
+    @FXML
+    private Pane aboutPopup;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources){
+    }
+
+    public void btnClosePopup() { hidePopup(); }
+
+    public void setTextAreaContent (String content){
+        textAreaPopup.setText(content);
+    }
+
+    public void hidePopup() { Platform.runLater(() -> aboutPopup.setVisible(false)); }
```

Comment:
not sure but I think this violates the codeStyle mentioned in https://github.com/Qabel/qabel-core/blob/master/CONTRIBUTING.md at least in terms of consistency.
put your method body in a new line and add a blank line between hidePopup and showPopup